pages/listings/listing/editListing.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="/dist/css/index.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Noroff Edit Job Listing</title>
+  </head>
+  <body class="d-flex flex-column flex-fill min-vh-100 gap-4" id="homepage">
+    <header></header>
+    <main class="container-fluid d-flex flex-fill flex-lg-row justify-content-center">
+      <div class="mt-5 mb-5" id="newListing" data-bs-backdrop="static" tabindex="-1">
+        <div class="border">
+          <!-- Waiting on API docs for validation requirements -->
+          <form class="m-2" id="createNewListing">
```

Comment:
Maybe change the ID to something like editLisiting, so the same ID isn't used again :)

Reply:
I'll fix it 👍